docs/framework/configure-apps/file-schema/configuration-sections-schema.md

```diff
@@ -22,33 +19,43 @@ helpviewer_keywords:
   - "configuration schema [.NET Framework], custom settings in configuration files"
   - "elements [.NET Framework], custom settings in configuration files"
 ms.assetid: 6e4cc793-c526-4007-b4e9-37d56295f2cb
-caps.latest.revision: 11
-author: "mcleblanc"
-ms.author: "markl"
-manager: "markl"
+author: "guardrex"
+ms.author: "mairaw"
+manager: "wpickett"
 ---
-# Configuration Sections Schema
-The configuration sections schema contains elements that put custom settings in configuration files.  
-  
- [\<configuration>](../../../../docs/framework/configure-apps/file-schema/configuration-element.md)  
-  
- [\<configSections>](http://msdn.microsoft.com/en-us/8a5cbc84-0257-4c2e-80a9-a064fe7c896b)  
-  
- [\<clear>](http://msdn.microsoft.com/en-us/b0ce3c91-89fa-4bb5-a995-30e21a61036b)  
-  
- [\<remove>](http://msdn.microsoft.com/en-us/6e2b2d45-113d-4d10-9f8e-0b15fc86bb98)  
-  
- [\<section>](http://msdn.microsoft.com/en-us/51c9d658-c275-4ab2-aa66-c5d5b5289570)  
-  
- [\<sectionGroup>](http://msdn.microsoft.com/en-us/45fbeb5c-26c7-4c68-b732-b3f1e62c53bf)  
-  
-|Element|Description|  
-|-------------|-----------------|  
-|[clear Element for configSections (General Settings Schema)](http://msdn.microsoft.com/en-us/b0ce3c91-89fa-4bb5-a995-30e21a61036b)|Clears all previously defined sections and section groups.|  
-|[configSections Element (General Settings Schema)](http://msdn.microsoft.com/en-us/8a5cbc84-0257-4c2e-80a9-a064fe7c896b)|Contains configuration section and namespace declarations.|  
-|[remove Element for configSections (General Settings Schema)](http://msdn.microsoft.com/en-us/6e2b2d45-113d-4d10-9f8e-0b15fc86bb98)|Removes a predefined section or section group.|  
-|[section Element for configSections (General Settings Schema)](http://msdn.microsoft.com/en-us/51c9d658-c275-4ab2-aa66-c5d5b5289570)|Contains a configuration section declaration.|  
-|[sectionGroup Element for configSections (General Settings Schema)](http://msdn.microsoft.com/en-us/45fbeb5c-26c7-4c68-b732-b3f1e62c53bf)|Defines a namespace for configuration sections.|  
-  
-## See Also  
- [Configuration File Schema](../../../../docs/framework/configure-apps/file-schema/index.md)
\ No newline at end of file
+
+# Configuration sections schema
+
+The configuration sections schema contains elements that define custom settings in configuration files.
```

Comment:
This is a longer list than the original document; none of the additions belong here. There also should be a TOC here that has Configuration Sections Schema with configSections as its child, etc.
It would also be good to figure out what headers to add here, other than See also -- which is the only thing that appears in the mini-TOC (not good).